### PR TITLE
fix: remove the dependency on git from the build script

### PR DESCRIPTION
**Pull request by nic-chen**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [x] Bugfix
- [ ] New feature provided
- [ ] Improve performance

___
### Bugfix
- Description

The source code package we will release does not contain git info, but the build script relies on the git command to obtain the git log version, which cause an error.

- How to fix?

remove the dependency on git from the build script

### CODE REVIEW

**Review by membphis — APPROVED** [review]
LGTM

**Comment by membphis** [comment]
rerun the CI ?

File: .github/workflows/release-test.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      - uses: actions/checkout@v2
+
+      - name: setup go
+        uses: actions/setup-go@v1
+        with:
+          go-version: '1.13'
+
+      - name: release and build
+        run: |
+          VERSION=$(cat ./api/VERSION)
```
Comment (juzhiyuan) [inline .github/workflows/release-test.yml:34]: This is a duplicate file like `Makefile`, could we use only one of them?

**Review by juzhiyuan — APPROVED** [review]
This CI would be improved in some ways.

**Pull request merged**